server/monitor/src/main/resources/org/apache/accumulo/monitor/templates/server.ftl

```diff
@@ -141,7 +141,7 @@
             <thead>
               <tr>
                 <th>Table&nbsp;</th>
-                <th>Tablet&nbsp;</th>
+                <th>Tablet&nbsp;(run shell cmd: getsplits -v)</th>
```

Comment:
This should be in the mouseover/tooltip, instead of in the column.

Reply:
I had thought of that but didn't think that it was obvious enough. Also, none of the columns on that table currently have tooltips.

Reply:
I just think that leaving it in the column header makes the interface messy, since it's instructions, and not part of the actual name/description of the field/column.

It's also a bit weird to specifically call out a separate tool without a more complete sentence, which we could do in a tooltip. The monitor and shell are largely independent tools that users could mix-and-match and replace with other tools, and different users might interface with each of them. So the monitor's explicit assumption that the user viewing the monitor has access to the Accumulo-provided shell and that they will understand "run shell cmd" as short for "run the Accumulo Shell client utility provided in the default Accumulo binary distribution tarball" are a little bit of a stretch for me.

Maybe the availability of a tooltip could be made more obvious with a little information symbol 🛈 or similar. I was thinking a longer tip, like `To associate the tablets listed here with their entries in the metadata table, use the <code>getsplits -v</code> command in the Accumulo Shell utility.`

Reply:
I like the information symbol idea. I will look and see if our web deps already have an icon included. The unicode you used isn't showing up for me

Reply:
From my comments on issue #1940 - would it be more useful if the column displayed the table-id and assigned server? Otherwise I agree with the tool-tip suggestion.

Reply:
I responded on #1940 but the short answer is no, it wouldn't make sense.